chocolatey/ReadMe.md

```diff
@@ -0,0 +1,132 @@
+﻿## Summary
```

Comment:
I assume this file is auto-generated by chocolatey?

If so I think it should be condensed to only include the necessary steps for building the Lektor package. If there is any more information of interest this could also go here.

Reply:
It is indeed auto-generated by chocolatey. I didn't know what to add in there so yes, I'll do the needful now.